### DISCUSSION TITLE
Client error: Gateway(DisallowedGatewayIntents)

### DISCUSSION THREAD

**Discussion by g-cyan** [serenity-rs/serenity#1529] [Q&A] [answered]
As I run ```cargo run``` , the following message is shown .

**Client error: Gateway(DisallowedGatewayIntents)**

What can I do for resolve it ?

**Comment by g-cyan** [comment, marked as answer]
![aa](https://user-images.githubusercontent.com/68693082/135784516-cd242343-43f2-4bf6-ac1d-d00c4eaf14dc.png)
As I checked both , the problem has been solved .

**Reply by JohnScience** [reply]
In my case I needed all three intents.
